voidcrew/modules/cloning/code/computer_cloning.dm

```diff
@@ -0,0 +1,627 @@
+#define AUTOCLONING_MINIMAL_LEVEL 3
+
+/obj/machinery/computer/cloning
+	name = "cloning console"
+	desc = "Used to clone people and manage DNA."
+	icon_screen = "dna"
+	icon_keyboard = "med_key"
```

Comment:
we need modular icons for this because TG might not have them anymore, or if they do they can delete them at any time. Nanites should do this, either at the moment or on the Science PR, you'll have to do the same here. Maybe we can add some common Computer dmi file for modular computers, because keyboard icons and such.